web-ng/src/app/services/dialog/dialog.service.spec.ts

```diff
@@ -17,17 +17,49 @@
 import { TestBed } from '@angular/core/testing';
 
 import { DialogService } from './dialog.service';
-import { MatDialogModule } from '@angular/material/dialog';
+import { MatDialog, MatDialogModule } from '@angular/material/dialog';
+import { ConfirmationDialogComponent } from '../../components/confirmation-dialog/confirmation-dialog.component';
 
 describe('DialogService', () => {
   let service: DialogService;
+  let dialogSpy: jasmine.Spy;
 
   beforeEach(() => {
     TestBed.configureTestingModule({ imports: [MatDialogModule] });
     service = TestBed.inject(DialogService);
+
+    dialogSpy = spyOn(TestBed.inject(MatDialog), 'open');
   });
 
   it('should be created', () => {
     expect(service).toBeTruthy();
   });
+
+  it('should return correct dialog without discard actions', () => {
+    service.openConfirmationDialog('testTitle', 'testMessage');
+
+    expect(dialogSpy).toHaveBeenCalledWith(ConfirmationDialogComponent, {
+      maxWidth: '500px',
+      autoFocus: false,
+      data: {
+        title: 'testTitle',
+        message: 'testMessage',
+        showDiscardActions: false,
+      },
+    });
+  });
+
+  it('should return correct dialog with discard actions', () => {
```

Comment:
If the public entry point to the class being tested is a single method I think it would be clearer to group tests by the method name `describe(..)`. In cases where multiple methods are tests at once we can adapt the description accordingly

Reply:
Okay, I changed the describe name! Let me know if this works now.